lib/resource.nunjucks

```diff
@@ -138,20 +138,26 @@
                             {% for key, item in b.formParameters %}
                               {% include "./item.nunjucks" %}
                             {% endfor %}
+
+                            {% if b.properties %}
+                              <ul>
+                                {% for key, item in b.properties %}
+                                  {% include "./item.nunjucks" %}
+                                {% endfor %}
+                              </ul>
+                            {% endif %}
+
                           </ul>
                         {% endif %}
 
                         {% if b.type and not b.schema %}
-                          <p><strong>Type</strong>:</p>
-                          {% if b.properties %}
-                            <ul>
-                              {% for key, item in b.properties %}
-                                {% include "./item.nunjucks" %}
+                          <p><strong>Type:</strong></p>
+                            <div class='list-group'>
+                              {% for typeName in b.type %}
+                                {% include "./type.nunjucks" %}
```

Comment:
Don't forget that URI parameters, headers, and query parameters are also type definitions in 1.0! The beauty to re-use the item.nunjucks template is that it could have been used for 0.8 and 1.0! Could you check if the type.nunjucks template works consistently for those nodes in 0.8 and 1.0?